cmake/modules/AddCephTest.cmake

```diff
@@ -1,5 +1,22 @@
 #AddCephTest is a module for adding tests to the "make check" target which runs CTest
 
+execute_process(
+    COMMAND uname -m
+    OUTPUT_VARIABLE ARCHSTR
+    OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+if(LINUX)
+  set(OSSTR linux)
+elseif(FREEBSD)
+  execute_process(
+    COMMAND uname -r
+    OUTPUT_VARIABLE OSVER
+    OUTPUT_STRIP_TRAILING_WHITESPACE)
+  set(OSSTR freebsd-${OSVER})
+endif(LINUX)
```

Comment:
@tchaikov 
Valid point.
I was more thinking about the startup trouble I had whilest starting porting.
But the code here is more than obvious to fix for new platforms.
I'll even throw in some free suggestions.